docs/contributors.md

```diff
@@ -22,17 +22,19 @@ make build-all
 export PATH=$(pwd)/bin:$PATH
 ```
 
-## Commands to build and load locally the kubeflex operator image for testing
+## Setting Up a Testing Cluster for KubeFlex
 
-```shell
-ko build --local --push=false -B ./cmd/manager -t $(git rev-parse --short HEAD) --platform linux/arm64
-kind load docker-image ko.local/manager:$(git rev-parse --short HEAD) --name kubeflex
-```
+To prepare a hosting cluster for testing, execute the following script. 
+This script accomplishes several key tasks:
 
-To deploy locally the image:
+- Creates a new kind cluster specifically designed for the KubeFlex hosting environment.
+- Configures nginx ingress with SSL passthrough capabilities to ensure secure communication.
+- Builds and loads the KubeFlex Controller Manager image into the kind cluster.
+- Installs a PostgreSQL database, providing the default backend for hosted API servers.
+- Starts the KubeFlex controller manager.
 
 ```shell
-make deploy IMG=ko.local/manager:$(git rev-parse --short HEAD)
+test/e2e/setup-kubeflex.sh 
```

Comment:
FYI, the omission of `source "${SRC_DIR}/setup-shell.sh"` here implies that the invocation of setup-kubeflex.sh can come earlier in test/e2e/run.sh (and if it had then I would not have had to manually verify lack of dependency).

Reply:
Addressed in last commit. Sourcing ${SRC_DIR}/setup-shell.sh only where required.

Reply:
I was only trying to suggest swapping two lines in `test/e2e/run.sh`, but OK.